Traceback (most recent call last):
  File "/home/miha/work/orange/testing/regressionTests/xtest_one.py", line 97, in <module>
    execfile(t__name)
  File "svm-easy.py", line 12, in <module>
    results = testing.cross_validation(learners, table, folds=5)
  File "/home/miha/work/orange/orange/Orange/misc/__init__.py", line 454, in wrap_call
    return func(*args, **kwargs)
  File "/home/miha/work/orange/orange/Orange/evaluation/testing.py", line 228, in cross_validation
    store_examples=store_examples)
  File "/home/miha/work/orange/orange/Orange/misc/__init__.py", line 454, in wrap_call
    return func(*args, **kwargs)
  File "/home/miha/work/orange/orange/Orange/evaluation/testing.py", line 295, in test_with_indices
    results, classifiers = self.one_fold_with_indices(learners, examples, fold, indices, preprocessors, weight)
  File "/home/miha/work/orange/orange/Orange/evaluation/testing.py", line 322, in one_fold_with_indices
    classifiers = [learner(learn_set, weight) for learner in learners]
  File "/home/miha/work/orange/orange/Orange/classification/svm/__init__.py", line 326, in __call__
    return self.learn_classifier(examples)
  File "/home/miha/work/orange/orange/Orange/classification/svm/__init__.py", line 516, in learn_classifier
    verbose=self.verbose))
  File "/home/miha/work/orange/orange/Orange/optimization/__init__.py", line 339, in __call__
    callCompareOn1st = True)
  File "/home/miha/work/orange/orange/Orange/misc/__init__.py", line 454, in wrap_call
    return func(*args, **kwargs)
  File "/home/miha/work/orange/orange/Orange/misc/selection.py", line 105, in __init__
    self.randomGenerator = random.Random(seed)
AttributeError: 'module' object has no attribute 'Random'
